packages/editor-ui/src/components/CodeNodeEditor/CodeNodeEditor.vue

```diff
@@ -150,20 +150,13 @@ export default defineComponent({
 	},
 	computed: {
 		...mapStores(useRootStore, usePostHog),
-		content(): string {
-			if (!this.editor) return '';
-
-			return this.editor.state.doc.toString();
-		},
 		aiEnabled(): boolean {
-			// AI is enabled only via Posthog experiment. So if user doesn't have feature flag
-			// we fallback to control which is no-show variant
-			const isAiExperimentDisabled =
-				(this.posthogStore.getVariant(ASK_AI_EXPERIMENT.name) ?? ASK_AI_EXPERIMENT.control) ===
-				ASK_AI_EXPERIMENT.control;
+			const isAiExperimentEnabled = [ASK_AI_EXPERIMENT.gpt3, ASK_AI_EXPERIMENT.gpt4].includes(
+				(this.posthogStore.getVariant(ASK_AI_EXPERIMENT.name) ?? '') as string,
```

Comment:
is `as string` needed?

Reply:
Yes, because `getVariant` return types are `boolean | string | undefined`
![CleanShot 2023-08-16 at 11 18 31](https://github.com/n8n-io/n8n/assets/12657221/d5a274b4-bd2f-428b-bcd6-91abd045eb82)

Reply:
ideally we would avoid type casting..